types/stripe-js/payment-intents.d.ts

```diff
@@ -531,6 +541,35 @@ export interface ConfirmBancontactPaymentData
   return_url?: string;
 }
 
+/**
+ * Data to be sent with a `stripe.ConfirmBlikPayment` request.
+ * Refer to the [Payment Intents API](https://stripe.com/docs/api/payment_intents/confirm) for a full list of parameters.
+ */
+export interface ConfirmBlikPaymentData extends PaymentIntentConfirmParams {
+  /**
+   * Either the `id` of an existing [PaymentMethod](https://stripe.com/docs/api/payment_methods), or an object containing data to create a `PaymentMethod` with.
+   * This field is optional if a `PaymentMethod` has already been attached to this `PaymentIntent`.
+   *
+   * @recommended
+   */
+  payment_method?: string | Omit<CreatePaymentMethodBlikData, 'type'>;
+
+  /**
+   * An object containing payment-method-specific configuration to confirm the [PaymentIntent](https://stripe.com/docs/api/payment_intents) with.
+   */
+  payment_method_options?: {
```

Comment:
According to the docs -  https://stripe.com/docs/js/payment_intents/confirm_blik_payment#stripe_confirm_blik_payment-data-payment_method_options-code 

the following are all required properties - `payment_method_options.blik.code` so you will need to remove the `?`

Reply:
Done. Additionally I've also changed `data` parameter to required in `confirmBlikPayment` declaration,  as it's required in the documentation.